src/handwritten/task/_task.py

```diff
@@ -233,7 +233,7 @@ def do_channels(self):
         return self._do_channels
 
     @property
-    def export_signals(self):
+    def export_signals(self) -> ExportSignals:
         """
         :class:`nidaqmx._task_modules.export_signals.ExportSignals`: Gets the
```

Comment:
Sphinx understands type hints. When you add type hints, you can remove the types from the docstring.

Also, regardless of type hints, we should make sure none of the docstrings refer to the old module names.